var/spack/repos/builtin/packages/wxPython/package.py

```diff
@@ -0,0 +1,47 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Wxpython(Package):
+    """A blending of the wxWidgets C++ class library with the Python
+    programming language."""
+
+    homepage = "http://www.wxpython.org/"
+    url = "http://downloads.sourceforge.net/wxpython/wxPython-src-3.0.2.0.tar.bz2"
+
+    version('3.0.2.0', '922b02ff2c0202a7bf1607c98bbbbc04')
+
+    depends_on('python')
+    depends_on('libpng')
+    depends_on('zlib')
+    depends_on('jpeg')
+    depends_on('libtiff')
```

Comment:
Those instructions build wxWidgets and Python (like a `+python` variant to the existing `wx` package). It appears (based on [Fedora's build](https://pkgs.fedoraproject.org/cgit/rpms/wxPython.git/tree/wxPython.spec)) that instead, the `wxPython/setup.py` should be used to build the Python bindings. This needs to do `setup.py WX_CONFIG=/usr/bin/wx-config-3.0 WXPORT=gtk3 build` to use the depended-on `wx`.